### ISSUE TITLE
update from 4.0 to 4.1.0 failed

### ISSUE THREAD

**Issue by zelphar** [YetiForceCompany/YetiForceCRM#4759]
#### Issue

import the update package 4.0 to 4.1.0v2
all page returns source code like this:
![image](https://user-images.githubusercontent.com/15126825/28996145-764e98c2-7a2c-11e7-95be-14786aa5c2ad.png)

#### How to trigger the error

1.
2.
3.

#### Your Environment

* YetiForce Version used:
* Browser name and version:
* Environment name and version:
* Operating System and version:

**Comment by mariuszkrzaczkowski** [comment]
I need more information. logs...

**Comment by zelphar** [comment]
since i upload files from complete package and overwrite the original files. the error disappeared.
i think its missing some file in update package.

**Comment by PercyP** [comment]
I tried to upload too and failed, but then I saw that 4.2 is being released today so going to try that. I can't do a fresh install as have done a lot of customisation which I am hoping I don't lose. 

When I did the upgrade test it said it was successful, afterwhich I could no longer access yetiforce - it kept returning me to my domain i.e. my yetiforce instance is at www.mysite.com/yetiforce and after the update, when I typed the url www.mysite.com/yetiforce it switched automatically to www.mysite.com. 

WHen I inspected it in CHrome there was an error 500, so I did a blanket fix on the files and folders but it still had no impact. I also noticed in the yetiforce folder there was a new directory called public_html with a number of files in there. Is this supposed to be there in the newer versions? There were no new errors in the cache/logs files.
I have reversed my test update version back to its previous state ready to test today's update - I will post back error logs if there are any etc.
